pkg/controller/tuned/status.go

```diff
@@ -45,11 +53,14 @@ func (r *ReconcileTuned) syncOperatorStatus() (bool, error) {
 		}
 	}
 	coState.Status.RelatedObjects = []configv1.ObjectReference{
-		{
-			Group:    "",
-			Resource: "namespaces",
-			Name:     dsManifest.Namespace,
-		},
+		{Group: "", Resource: "namespaces", Name: tunedManifest.Namespace},
```

Comment:
There are a number of `ObjectReference`s without groups like this one. I'm kinda surprised that's the case.

Reply:
Empty group "" indicates the core API group afaik.  In any case, tested and worked fine.